packages/core/src/models/promptVariants.ts

```diff
@@ -90,24 +94,29 @@ export class PromptVariants {
     });
   }
 
-  format() {
+  format(raw = false) {
```

Comment:
Might be helpful to have a type annotation here, possible passing arg as an object? Thinking that seeing the invocation somewhere `format(true)` isn't clear